interpreter.py

```diff
@@ -198,6 +236,10 @@ def exec_operation(self, instruction):
             var_val = self.lookup_var(args[0])
             assert isinstance(var_val, Float)
             return Float(var_val.value)
+        elif opcode == LLVMZExt:
+            var_val = self.lookup_var(args[0])
+            assert isinstance(var_val, Integer)
+            return Integer(var_val.value)
```

Comment:
Yep, but I called them "mostly immutable" because I was a bit unsure whether I ever modify the value of an existing `Integer`, because I sometimes modify values of existing variables. I think they are not immutable because my variable dictionary stores wrapper values, such as `Integer` or `Ptr`, which I sometimes alter with set_var.